webpack.config.js

```diff
@@ -1,9 +1,54 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+const extractSASS = new ExtractTextPlugin('[name].css');
+
 module.exports = {
-  context: __dirname + "/app",
-  entry: "./app.js",
+  context: path.join(__dirname, '/src/model'),
+  entry: { 
+    calculator: './index.js',
+    //test: "./index.test.js"
+  },
 
   output: {
-    filename: "app.js",
-    path: __dirname + "/dist",
+    path: path.join(__dirname + '/public'),
+    filename: '[name].bundle.js',    
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        use: [
+          { loader: 'style-loader' },
+          { loader: 'css-loader' }
+          //{ loader: 'postcss-loader' }
+        ]
+      },
+      {
+        test: /\.scss$/,
+        use: extractSASS.extract({
+          fallback: 'style-loader',
+          use: ['css-loader', 'sass-loader']
+        })
+        // use: [
+        //   { loader: 'style-loader' },
+        //   { loader: 'css-loader' },
+        //   { loader: 'sass-loader' }
```

Comment:
if that's the case, I would rather create two objects for each of the use case, then reference them as needed
something like:
```js
use: extractSassLoader,
// use: defaultSassLoader,
```